src/Aquarius.ONE.ClientSDK/Common/Logbook/LogbookApi.cs

```diff
@@ -112,12 +112,31 @@ public async Task<bool> DeleteLogbookAsync(string logbookId)
         /// <param name="logbookId">Identifier of the logbook in which to create the entry</param>
         /// <param name="entry">Text of the entry to be created</param>
         /// <param name="entryTime">Timestamp to be associated to the entry, should be in UTC</param>
+        /// <param name="geoPointX">Optional - the longitude of the location of this entry</param>
+        /// <param name="geoPointY">Optional - the latitude of the location of this entry</param>
         /// <param name="tags">Any tags that should be associated to the entry, no spaces allowed</param>
         /// <returns>Boolean indicating whether or not the logbookEntry was successfully created</returns>
-        public async Task<bool> CreateLogbookEntryAsync(string logbookId, string entry, DateTime entryTime, params string[] tags)
+        public async Task<bool> CreateLogbookEntryAsync(string logbookId, string entry, DateTime entryTime, float? geoPointX = null, float? geoPointY = null, params string[] tags)
```

Comment:
Don't you also need to know the datum in which the lat/long is recorded in (E.G. [WSG84](https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84)? Otherwise if someone tries to plugin in a lat/long from another coordinate system/datum, your point won't plot on the map correctly.

I suppose this could be resolved by simply stating the geoPointX/Y is in some hard coded datum, but it might be good to explicitly state that.

I'm not familiar with all this so it might be documented somewhere else.

Reply:
It's been a while since I've done any geography stuff but I think you are right we need to either assume a specific one or ask for it to be provided.

Reply:
Looking at the point2D protobuf there is not any datum property so perhaps we are assuming one?  https://github.com/AquaticInformatics/ONE.Interfaces.ProtocolBuffers/blob/develop/proto/one/common/core/point2d.proto

Reply:
Dan is correct that there's not currently a path to providing this from the client.  Currently, it is hard coded to WGS 84,  I'm up for a discussion about how to make this more flexible for customers as we take over the world, but for our current customers I feel that this should be reasonably accurate.